libs/ballot-interpreter/src/legacy_adapter.ts

```diff
@@ -99,47 +177,68 @@ function convertNewHampshireNextMarkToSharedMark(
   throwIllegalValue(option);
 }
 
+function convertScoredContestOptionsToMarkInfo(
+  geometry: Geometry,
+  scoredContestOptions: ScoredContestOption[]
+): MarkInfo {
+  const markInfo: MarkInfo = {
+    ballotSize: geometry.canvasSize,
+    marks: scoredContestOptions.map((scoredContestOption) =>
+      convertScoredContestOptionToLegacyMark(scoredContestOption)
+    ),
+  };
+
+  return markInfo;
+}
+
+function getUnmarkedWriteInsFromScoredContestOptions(
+  scoredContestOptions: ScoredContestOption[]
+): InterpretedHmpbPage['unmarkedWriteIns'] {
```

Comment:
I noticed there are a number of places that you use `InterpretedHmpbPage['unmarkedWriteIns']` rather than using whatever type it is directly. Is there a reason for that? Curious because it's different from what I usually see, so when I see it I usually think there's some meaning to it

Reply:
Not a strong reason, although sometimes I prefer it because it communicates more. It indicates that the UWI type is a component of the interpretation type, which tells me a little something about how to find it and its purpose.